MapboxMobileEvents/MMEConstants.m

```diff
@@ -25,14 +27,16 @@
 NSString * const MMEDebugEventTypeBackgroundTask = @"backgroundTask";
 NSString * const MMEDebugEventTypeMetricCollection = @"metricCollection";
 NSString * const MMEDebugEventTypeLocationManager = @"locationManager";
+NSString * const MMEDebugEventTypeTelemetryMetrics = @"telemMetrics";
```

Comment:
What's the difference between `telemMetrics` and `telemetryMetrics`? I'm confused by that.

Reply:
`telemMetrics` is only used for internal debugging purposes. `telemetryMetrics` is used when posting the actual event to the server.